ghost/core/core/server/data/schema/schema.js

```diff
@@ -985,7 +985,9 @@ module.exports = {
         started_at: {type: 'dateTime', nullable: true},
         finished_at: {type: 'dateTime', nullable: true},
         created_at: {type: 'dateTime', nullable: false},
-        updated_at: {type: 'dateTime', nullable: true}
+        updated_at: {type: 'dateTime', nullable: true},
+        metadata: {type: 'text', maxlength: 1000000000, fieldtype: 'long', nullable: true},
+        queue_entry: {type: 'integer', nullable: true, unsigned: true}
```

Comment:
Just curious: why integer instead of string?

Also a small nit: would `queue` suffice as a column name?

Reply:
I felt like the schema changes were the most contentious 😂.

Right now it's being used to indicate a queue entry when `queue_entry = 1`. We could do various queues, but that would be a departure from the design of the current queue, which is a 'FIFO for all things' kind of model. I'm not sure I want to get into the complications of multiple queues without having proofed this on Pro, or seen that it's necessary.

Indexing is faster w/ integer values, I believe, and I was thinking of just making this 0/1. (I also considered adding an index to this for `queue,created_at` if we think it'll get big/slow).

I was also wondering if instead of multiple queues by string values, if we wanted to make priorities ie. when pulling jobs, `order by queue_entry,created_at desc`. 

Wdyt?

Reply:
> I felt like the schema changes were the most contentious 
Ha, yeah definitely — it's easy enough to change everything else down the line but I think it's worth putting some thought into making sure we get the schema as close to correct as we can the first time.

Yeah keeping it as an int makes sense for now I think since we're not doing multi-queues right away, but I'd like to redact my nit about the name, and keep it as `queue_entry` so that we can still use a string `queue` field in the future if we go down that route.

I think long-term we should move toward _all_ jobs in this table being in the queue, so eventually we can even remove this field altogether. But I think this makes sense given that this table is currently a bit mixed-use.

Reply:
Yep, agreed. Still not 100% on how we'd want to handle scheduled jobs in that sort of setup.